deploy/utils/backup_script/run_backup.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+LOCKFILE=/tmp/backup_script.lock
+
+if [ -e "$LOCKFILE" ]; then
+    echo "Script is running now. Lockfile enabled."
+    exit 1
+fi
+
+touch $LOCKFILE
+
+source /root/.bashrc
+
+script_dir="/sportchain/backup_script"
+log_dir="$script_dir/logs"
+
+mkdir -p $log_dir
+
+log_file="$log_dir/backup_$(date +%F_%H-%M-%S).log"
+
+exec > >(tee -a "$log_file") 2>&1
+
+max_logs=23
+log_count=$(ls -1q "$log_dir" | wc -l)
+
+if [ "$log_count" -gt "$max_logs" ]; then
+    oldest_log=$(ls -t "$log_dir" | tail -n 1)
+    echo "Deleting oldest log: $oldest_log"
+    rm "$log_dir/$oldest_log"
+fi
+
+current_date=$(date +%F_%H-%M-%S)
+
+block=$(curl -s -H "Content-Type: application/json" -d '{"id":1, "jsonrpc":"2.0", "method": "chain_getBlock", "params":[]}' http://185.190.140.207:9944 | jq -r '.result.block.header.number')
```

Comment:
Can we use domain name instead of IP?

Reply:
oh, i've just noticed you used ip of the devnet. please, pass the ip as argument or via config/.env file. and as the testnet GENESIS machine doesn't expose the port, you need to use the localhost.

to @hrls no, the machine where we backing up doesn't have a dedicated domain name.